cmr/queries.py

```diff
@@ -668,7 +742,7 @@ def __init__(self, mode: str = CMR_OPS):
         self.concept_id_chars = {"G", "C"}
 
     def orbit_number(
-        self, orbit1: FloatLike, orbit2: Optional[FloatLike] = None
+            self, orbit1: FloatLike, orbit2: Optional[FloatLike] = None
```

Comment:
I think extra indentation slipped in via another PR:

```suggestion
        self, orbit1: FloatLike, orbit2: Optional[FloatLike] = None
```